09_json/n0npax/pkg/puppy/types.go

```diff
@@ -0,0 +1,17 @@
+package puppy
+
+// Puppy contains information about single puppy
+type Puppy struct {
+	ID     int    `json:"id"`
+	Value  int    `json:"value"`
```

Comment:
Value could be 1,99... (float) :laughing:

Reply:
design assumption :)